src/main/java/com/app/onlinebookstore/controller/BookController.java

```diff
@@ -24,7 +25,7 @@ public class BookController {
     private final BookService bookService;
 
     @PostMapping
-    public BookDto createBook(@RequestBody CreateBookRequestDto bookDto) {
+    public BookDto createBook(@RequestBody @Valid CreateBookRequestDto bookDto) {
         return bookService.save(bookDto);
```

Comment:
you forgot to add validation when updating book

Reply:
Agree

Reply:
Add this in the next Pr

Reply:
add to this pr, it's not much work